TAO/tao/Strategies/advanced_resource.cpp

```diff
@@ -186,95 +176,80 @@ TAO_Advanced_Resource_Factory::init (int argc, ACE_TCHAR** argv)
 #endif  /* ACE_HAS_EVENT_POLL || ACE_HAS_DEV_POLL */
             }
 
-          else if (ACE_OS::strcasecmp (current_arg,
-                                       ACE_TEXT("fl")) == 0)
-            this->report_option_value_error (
-                ACE_TEXT("FlReactor not supported by Advanced_Resources_Factory. Please use TAO_FlResource_Loader instead."),
-                         current_arg);
-          else if (ACE_OS::strcasecmp (current_arg,
-                                       ACE_TEXT("tk")) == 0)
-            this->report_option_value_error (
-                ACE_TEXT("TkReactor not supported by Advanced_Resources_Factory. Please use TAO_TkResource_Loader instead."),
-                         current_arg);
+          else if (ACE_OS::strcasecmp (current_arg, ACE_TEXT ("fl")) == 0)
+            this->report_option_value_error (ACE_TEXT ("FlReactor not supported by Advanced_Resources_Factory. Please use TAO_FlResource_Loader instead."),
+                                             current_arg);
+          else if (ACE_OS::strcasecmp (current_arg, ACE_TEXT ("tk")) == 0)
+            this->report_option_value_error (ACE_TEXT ("TkReactor not supported by Advanced_Resources_Factory. Please use TAO_TkResource_Loader instead."),
+                                             current_arg);
+          else if (ACE_OS::strcasecmp (current_arg, ACE_TEXT ("single_input")) == 0)
+            this->reactor_type_ = TAO_REACTOR_SINGLE_INPUT;
```

Comment:
Mention in docs

Reply:
I updated TAO\docs\Options.html to add this new `-ORBReactorType ` type. Is there other place that needs to be updated?

Reply:
No, that is the place